source/clear-linux/get-started/install-configfile.rst

```diff
@@ -145,29 +151,28 @@ Perform the following steps:
 
    .. code-block:: bash
 
-   	  # cd /root
-   	  # nano clr-installer.yaml
+   	  cd /root
+   	  nano clr-installer.yaml
 
 #. Copy the :file:`clr-installer.yaml` file, replacing <file-path> with the
    desired location on the other target.

```

Comment:
Ln 158: I apologize that I missed this before. I recommend removing "other" here.  "Target" alone is fine.